packages/upgrade/test/__fixtures__/hooks/existing-rq.ts

```
@@ -0,0 +1,6 @@
+import { useMutation, useQuery } from '@tanstack/react-query';
```

ESLint

[error] Filename is not in camel case. Rename it to `existingRq.ts`.

(unicorn/filename-case)